<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersPage</class>
 <widget class="QWizardPage" name="UsersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="title">
   <string>Elèves</string>
  </property>
  <property name="subTitle">
   <string>Sélectionner les élèves qui auront des khôlles dans chaque matière</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Semaine courante :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Semaine paire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Semaine impaire</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date du lundi :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
